### ISSUE TITLE
Add top k metric

### ISSUE THREAD

**Issue by joclement** [joclement/master_thesis#44]
See [here](https://scikit-learn.org/0.24/modules/generated/sklearn.metrics.top_k_accuracy_score.html#sklearn.metrics.top_k_accuracy_score).
